subservers/manager.go

```diff
@@ -259,3 +288,22 @@ func dialBackend(name, dialAddr, tlsCertPath string) (*grpc.ClientConn, error) {
 	}
 	return cc, nil
 }
+
+// readMacaroon tries to read the macaroon file at the specified path and create
+// gRPC dial options from it.
```

Comment:
Does `...and create gRPC dial options from it.` apply here?

Reply:
good catch!

Reply:
fixed